charts/app-config/values-integration.yaml

```
@@ -429,18 +429,17 @@ govukApplications:
   helmValues:
     ingress:
       enabled: true
+      redirect: true
       annotations:
         <<: *default-alb-ingress-annotations
         <<: *alb-ingress-backend-waf-ruleset
         alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: >
           [{"field": "host-header", "hostHeaderConfig": { "values": [
-              "content-data.{{ .Values.publishingDomainSuffix }}",
-              "content-data-admin.{{ .Values.publishingDomainSuffix }}"
+              "content-data.{{ .Values.publishingDomainSuffix }}"
           ]}}]
-        alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": {
-          "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
-        alb.ingress.kubernetes.io/actions.{{ .Release.Name }}: '{"Type":"redirect","RedirectConfig": {
-          "Host":"content-data.{{ .Values.k8sExternalDomainSuffix }}","Path":"/#{path}","Port":"443","Protocol":"HTTPS","Query":"#{query}","StatusCode":"HTTP_301"}}'
+        alb.ingress.kubernetes.io/actions.{{ .Release.Name }}-redirect: '{"Type":"redirect","RedirectConfig":{ "Host":"content-data.{{ .Values.publishingDomainSuffix }}","Path":"/#{path}","Port":"443","Protocol":"HTTPS","Query":"#{query}","StatusCode":"HTTP_301"}}'
```

optional: could use a multiline string to format this a bit more readably, something like:

```suggestion
        alb.ingress.kubernetes.io/actions.{{ .Release.Name }}-redirect: >
          {
            "Type":"redirect","RedirectConfig": {
              "Host":"content-data.{{ .Values.publishingDomainSuffix }}",
              "Path":"/#{path}",
              "Port":"443",
              "Protocol":"HTTPS",
              "Query":"#{query}",
              "StatusCode":"HTTP_301"
            }
          }
```